lib/sensors/include/URM04Sensor.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+// includes
+#include "BufferedSerial.h"
+#include "Thread.h"
+#include "mbed.h"
+#include "rtos.h"
+
+// constants
+#define urmAccount 1
+#define MAX_TRIES  40
```

Comment:
this seems highly excessive. do you expect this to fail often? if we call read() it should try once. in ur test driver you should measure how long read() takes. if it's rly quick then _maybe_ try twice

Reply:
This is just from the arduino code but I'll reduce the number of tries